packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/virtualScroll.js

```diff
@@ -1,8 +1,17 @@
-export default function virtualScroll(x, y, remove) {
-  const w = x === 0 ? 0 : -1 * x;
-  const h = y === 0 ? 0 : -1 * y;
+/**
+ * Script to apply a negative 2D-translate transformation method to move the document from its current position
+ * to up and to the right for the specified number of pixels horizontally and vertically.
```

Comment:
I wasn't sure what to make of this. "...current position to up and to the right...".  Should it just be "...current position to the specified..."

Reply:
yeah this one was weird. was trying to caputre that this changes the value to the negative x or y value provided